### DISCUSSION TITLE
Why doesn’t @dbt_assets have group_name?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17830] [Q&A] [answered]
Hey Everyone! Curious as to why @dbt_assets() doesn’t have group_name on it so you can put it in an asset_group

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699468653869029?thread_ts=1699468653.869029&cid=C01U954MEER).*

**Comment by tacastillo** [comment, marked as answer]
Hi! If you'd like to organize your dbt models in groups, you can either:
* use dbt's official `group` property to organize your models
* define the groups in the `meta` for a dbt model
* or you can override them and have Dagster manage them by overriding `DagsterDbtTranslator`'s `get_group_name` method.

Both ways are covered in this section of our docs. https://docs.dagster.io/integrations/dbt/reference#customizing-group-names


Cheers!

**Reply by magidandrew** [reply]
I got stuck on this for a while...dagster pulls out the groupings out of the manifest.json

So even if you add these properties (`group` or `meta`) to your models, if you don't run `dbt compile`, then you will not see the asset groups change in the dagit ui

save yourself an hour, hope this helps!
